chainio/clients/builder.go

```diff
@@ -27,6 +27,7 @@ type BuildAllConfig struct {
 	OperatorStateRetrieverAddr string
 	AvsName                    string
 	PromMetricsIpPortAddress   string
+	RewardsCoordinatorAddress  string
```

Comment:
why add the `RewardsCoordinator` here?

Reply:
The `RewardsCoordinatorAddress` is part of the [elcontracts.Config](https://github.com/Layr-Labs/eigensdk-go/blob/dev/chainio/clients/elcontracts/reader.go#L25), and its used when [building the ElContracts Clients](https://github.com/Layr-Labs/eigensdk-go/blob/dev/chainio/clients/elcontracts/builder.go). Anyways, its was needed here to test the rewards methods, but we decided not to add those tests in this PR, so I could remove this for now.